src/components/menu-item/menu.stories.tsx

```diff
@@ -99,7 +98,7 @@ const Template: StoryFn = ( args ) => (
 	</Menu>
 );
 
-export const CombinedMenu = Template.bind( {} );
+export const CombinedMenu: Story = Template.bind( {} );
 
 CombinedMenu.args = {
 	size: 'md',
```

Comment:
Show arrow on hover not working? When `showArrowOnHover = false` it does not show the icon.

Reply:
this setting only for showing on hover. It stops on hover based on the boolean.